src/nft/components/card/containers.tsx

```diff
@@ -186,22 +186,22 @@ const Container = ({
   isSelected,
   isDisabled,
   detailsHref,
-  doNotLinkToDetails = false,
+  linkToDetails = true,
   testId,
   onClick,
   children,
 }: {
   isSelected: boolean
   isDisabled: boolean
   detailsHref: string
-  doNotLinkToDetails: boolean
+  linkToDetails: boolean
   testId?: string
   children: ReactNode
   onClick?: (e: React.MouseEvent) => void
 }) => {
   return (
     <CardContainer isSelected={isSelected} isDisabled={isDisabled} testId={testId} onClick={onClick}>
-      <StyledLink to={doNotLinkToDetails ? '' : detailsHref}>{children}</StyledLink>
```

Comment:
We should also get rid of `detailsHref` entirely. This should just be a simple UI component.

Reply:
PS. Once you get rid of the two aforementioned props, you can just render a direct `CardContainer` instead (a Styled Component), you do not need a wrapper, because these props will be automatically passed to an underlying component!